src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.DragDrop.cs

```diff
@@ -153,11 +153,13 @@ private static void OnItemContainerDragStarting(UIElement sender, DragStartingEv
 					that.DragLeave -= OnReorderDragLeave;
 					that.Drop -= OnReorderCompleted;
 
-					that.DragEnter += OnReorderDragUpdated;
-					that.DragOver += OnReorderDragUpdated;
-					that.DragLeave += OnReorderDragLeave;
-					that.Drop += OnReorderCompleted;
-
+					if (that is not Microsoft.UI.Xaml.Controls.TreeViewList)
```

Comment:
Could we check something more meaningful like an internal flag `IsCustomReorder` (or any other name ...) ?

Reply:
Done